bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/thing/security.xml

```diff
@@ -10,6 +10,12 @@
 			<channel id="person-count" typeId="person-count"/>
 			<channel id="unknown-person-count" typeId="unknown-person-count"/>
 			<channel id="unknown-person-picture" typeId="unknown-person-picture"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="security-event">
+		<label>Home Security</label>
```

Comment:
This label should probably be added to the default translations properties.

Reply:
fixed